avocado/utils/lv_utils.py

```diff
@@ -617,6 +617,33 @@ def lv_mount(vg_name, lv_name, mount_loc, create_filesystem=""):
     except process.CmdError as ex:
         raise LVException("Fail to mount logical volume: %s" % ex)
 
+def vg_reactivate(vg_name, timeout=10, export=False):
+    """
+    In case of unclean shutdowns some of the vgs is still active and merging
+    is postponed. Use this function to attempt to deactivate and reactivate
+    all of them to cause the merge to happen.
+
+    :param str vg_name: name of the volume group
+    :param int timeout: timeout between operations
+    :raises: :py:class:`LVException` if the logical volume is still active
+    """
+    try:
+        process.run("vgchange -an %s" % vg_name, sudo=True)
+        time.sleep(timeout)
```

Comment:
any reason why we have use  large sleep value as not looking good to me can this will be reduced as we use other places `.1` and also  no need to expect as function argument

Reply:
@PraveenPenguin This is the way it is handled in the lv_reactivate also, from where this function is inspired from :-)

The reason for large sleep value is, sometimes these functions will take longer in larger volumes full of data.

Reply:
ah sure , than will merge